src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -127,13 +127,21 @@
             "group size would be approximately 10 reads.")
     public int MAX_GROUP_RATIO = 500;
 
+    @Option(doc = "Barcode SAM tag (ex. BC)", optional=true)
```

Comment:
If you are trying to refer to the 10X Genomics barcode in this doc line you should use BX or RX not BC. The BC field in our bams is the sample_index. The RX field is the raw barcode. BX is the corrected and validated (must be in barcode whitelist we expect) barcode as well as gem_group attached (if you do multiple library preps they cannot be dups of one another and did not come from the same long molecules).